sentinel-adapter/sentinel-web-servlet/src/main/java/com/alibaba/csp/sentinel/adapter/servlet/CommonFilter.java

```diff
@@ -107,7 +107,9 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
             if (entry != null) {
                 entry.exit();
             }
-            ContextUtil.exit();
+            if (ContextUtil.getContext() != null) {
```

Comment:
This judgment seems unnecessary, since it has been done inside `ContextUtil.exit()` method.